views/artist-search-results.hbs

```diff
@@ -0,0 +1,17 @@
+  <div class="artists">
+      
+  </div>{{#each findArtists}}
```

Comment:
in the app.js file you are passing the data to the view with the name results, so in the view you need to say {{#each result}}

if you prefer you can rename it in the app.js file to ` res.render("artist-search-results", { findArtists: results });
`